src/main/java/frc/bionic/TrajectoryUtils.java

```diff
@@ -26,24 +27,35 @@ public static Trajectory generateTrajectory(Pose2d startPos,
                                             double rotation, 
                                             AbstractDrivetrain drivetrain, 
                                             Translation2d... interiorWaypoints) {
-      // Creates a new Trajectory Config, which stores Max Velocity, Max Acceleration and Kinematics
-      TrajectoryConfig config =
-            new TrajectoryConfig(
-                    // Max Velocity Meters Per Seconds
-                    Conversion.inchesToMeters(120),
-                    // Max Acceleration Meters Per Seconds Squared
-                    Conversion.inchesToMeters(144))
-                // Add kinematics to ensure max speed is actually obeyed
-                .setKinematics(drivetrain.getKinematics());
-          //Updates Shuffleboard
+            // Creates a new Trajectory Config, which stores Max Velocity, Max Acceleration and Kinematics
+            TrajectoryConfig config =
+                  new TrajectoryConfig(
+                        // Max Velocity Meters Per Seconds
+                        Conversion.inchesToMeters(120),
+                        // Max Acceleration Meters Per Seconds Squared
+                        Conversion.inchesToMeters(144))
+                  // Add kinematics to ensure max speed is actually obeyed
+                  .setKinematics(drivetrain.getKinematics());
 
-      Pose2d endPos = new Pose2d(endX, endY, Rotation2d.fromDegrees(rotation));
-      Trajectory returnTrajectory = TrajectoryGenerator.generateTrajectory(startPos, 
-                                                                            Arrays.asList(interiorWaypoints), 
-                                                                            endPos, 
-                                                                            config);
-      return returnTrajectory;/*.transformBy(
-                                      new Transform2d(drivetrain.getCurrentPose().getTranslation(), 
-                                      Rotation2d.fromDegrees(0))); */
+            Pose2d endPos = new Pose2d(endX, endY, Rotation2d.fromDegrees(rotation));
+            Trajectory returnTrajectory = TrajectoryGenerator.generateTrajectory(startPos, 
+                                                                              Arrays.asList(interiorWaypoints), 
+                                                                              endPos, 
+                                                                              config);
+            return returnTrajectory;
+      }
+
+      public static Trajectory generateSCurve(TrajectoryConfig config){
```

Comment:
This looks like a testing function. It's likely fine to keep it in Utils for future reference and testing, but it should be documented as for that purpose. (Unless it's for some other purpose, in which case, it still should have documentation as to its purpose.)